### PR TITLE
Updated the icons for the plex, region, and CICS profile

### PR DESCRIPTION
**Pull request by Rajat-Sahrawat**
6 icons for CICS profile (connected, disconnected, unverified) - light and dark
4 icons for region (connected, disconnected)- light and dark
2 for plex- light and dark

### CODE REVIEW

File: packages/vsce/resources/imgs/profile-dark.svg
Comment (davenice) [inline packages/vsce/resources/imgs/profile-dark.svg:1]: this icon is 162px x 150px - can we keep it square?

**Review by davenice — CHANGES_REQUESTED** [review]
hey @Rajat-Sahrawat - thanks for these! we'll need to get your commits signed (that's the DCO check) - we can work together on that.

i think we should aim to keep them 150px square like the originals - how feasible is it to do that?

Here are some screenshots of what they look like within the GUI:
image

image

image

I think these look good - I like the plex and region icons especially.

@Joe-Winchester might have thoughts too!

**Comment by davenice** [comment]
on reflection - before asking @Rajat-Sahrawat to fiddle around with the aspect ratios... @JillieBeanSim / @zFernand0 i see you've been requested to review - is there any requirement to stick with square? it appeals to my sense of tidiness, but.... :-)

File: packages/vsce/resources/imgs/profile-dark.svg
Comment (Rajat-Sahrawat) [reply packages/vsce/resources/imgs/profile-dark.svg:1]: Changed all the icons to 300x300 px

**Comment by Rajat-Sahrawat** [comment]
fixed some size issues, signed and updated. @zFernand0 @JillieBeanSim ready for review now

**Comment by Rajat-Sahrawat** [comment]
@zFernand0 @JillieBeanSim Let me know if I need to update the change log for the new icons

**Review by zFernand0 — APPROVED** [review]
The new icons look great! 🙏 
Thanks @Rajat-Sahrawat 
And thanks @davenice  for posting pictures of them 🙏

**Pull request merged**